pyaerocom/io/readungridded.py

```diff
@@ -486,8 +488,12 @@ def read_dataset(
         UngriddedData
             data object
         """
+        force_caching = False
```

Comment:
yes in the config we have 
```
OBS_CONFIG["EEA"] = dict(
    obs_id="CAMS2_83.NRT",
    # obs_id="EEAAQeRep.NRT",
    obs_vars=species_list,
    web_interface_name="EEA-NRT",
    obs_vert_type="Surface",
    read_opts_ungridded=dict(files=[], force_caching=True),
    obs_filters=obs_filters,
)
``` 
and since the latest changes in `EvalSetup` I think we have a bug :  
the .pkl files with the obs are correctly written when the standard run starts, but then when I run the medianscores the cache appears to be empty even though it is explicitly not cleared, so now that you mention this, we might indeed be missing something

Reply:
I checked and the observations are cached as they should, I have looked into the way the new code from main-dev handles cache and I conclude that the new RM_CACHE_OUTDATED env variable, True by default, is what is clearing the cache before the median scores run despite the cams2_83 cli being set to skip the clearing in that case

so forget what I wrote above, it's not a bug, I just need to set that variable to false for cams2_83